plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/AppEngineProjectDeployer.java

```diff
@@ -0,0 +1,88 @@
+package com.google.cloud.tools.eclipse.appengine.deploy;
+
+import java.io.File;
+import java.util.Collections;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.core.runtime.SubMonitor;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+
+import com.google.cloud.tools.appengine.api.deploy.DefaultDeployConfiguration;
+import com.google.cloud.tools.appengine.cloudsdk.CloudSdk;
+import com.google.cloud.tools.appengine.cloudsdk.CloudSdkAppEngineDeployment;
+
+public class AppEngineProjectDeployer {
+
+  private static final String PLUGIN_ID = "com.google.cloud.tools.eclipse.appengine.localserver"; //$NON-NLS-1$
+  private static final String APPENGINE_WEB_XML_PATH = "WEB-INF/appengine-web.xml"; //$NON-NLS-1$
+
+  public void deploy(IPath stagingDir, CloudSdk cloudSdk, IProgressMonitor monitor) throws CoreException {
+    SubMonitor progress = SubMonitor.convert(monitor, 1);
```

Comment:
My reasoning here was that if there's not internal steps to update the monitor for, I could go with any arbitrary number of ticks. If 100 feels more intuitive (because of 100%), I'm happy to change it.
Do I miss any other piece that would influence the number of ticks here?